src/plasmapy/simulation/particle_tracker/particle_tracker.py

```diff
@@ -681,6 +711,15 @@ def run(self) -> None:
 
             self._push()
 
+            desc = (
```

Comment:
More printout to help the user track the progress of the run. Progress bar is hard for this application since there isn't a definite number of iterations, so all this info is very useful. Also for debugging if you set it up wrong.